setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name='stempeg',
-    version='0.1.8',
+    version='0.2.0',
```

Comment:
There's a ref to Python 2.7 remaining in your classifiers but I think you're no longer compatible